other/copy-namespace-labels/.chainsaw-test/resource.yaml

```diff
@@ -0,0 +1,85 @@
+#Deployment with Owner Label and additional labels
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-1
+  namespace: within-ns
+  labels:
+    app: good-app    
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: good-app
+  template:
+    metadata:
+      labels:
+        app: good-app
+    spec:
+      containers:
+      - name: good-app-deploy
+        image: busybox:1.35
+---
+#Deployment with `kubernetes.io/metadata.name` label
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-2
+  namespace: within-ns
+  labels:
+    app: my-app02
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: my-app02
+  template:
+    metadata:
+      labels:
+        app: my-app02
+    spec:
+      containers:
+      - name: my-app02-deploy
+        image: busybox:1.35
+---
+#Deployment with only `owner` label
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: my-deployment-3
+  namespace: within-ns
+  labels:
+    owner: "any-corp"
```

Comment:
You should change the value of this label and ensure it is overwritten with the label's value from the Namespace. The patched resource for this is fine, but change the value of `owner` here to something different.